### DISCUSSION TITLE
Is there a way to see when saving a source member, that the save is done?

### DISCUSSION THREAD

**Discussion by dhfreinkel** [codefori/vscode-ibmi#788] [Q&A]
I have a problem when I save a member, I never know if the save has completed or not.

Can we get a message displayed to save the save completed like the message when a compile completes?

TIA, Darryl Freinkel.

**Comment by worksofliam** [comment]
There actually in an indicator in the file tab. When the file is dirty (edited), then a circle appear on the tab (where the close X is). When the file is saved, it is no longer dirty, it returns to the standard clone button.

It is not likely I will add a notification to show a file has saved.

**Comment by dhfreinkel** [comment]
I thought the dot was there. I have occasionally seen it, but I have not seen it for a number of weeks.

 

The message would help so I know when I can compile the program.

 

Darryl Freinkel

A4G

Telephone: 770.321.8562 Mobile: 678.355.8562
